src/summaries/decentralized_exchanges.yaml

```diff
@@ -0,0 +1,128 @@
+        * Trade settlement might be slow due to blockchain scalability issues. 
+        * Sometimes, not fully decentralized due to mediating server for speeding the process of trade execution.
+
+
+- title: Automated Market Maker (AMM)
+  shortTitle: AMM
+  key: amm
+  details: |
+    - Automated Market Maker (AMM):
+      * AMM is a type of DEX protocol that relies on a mathematical formula that allows digital assets to be traded without permission, automatically by using liquidity pools instead of the traditional order book pricing model. This formula can vary with each protocol. For example, Uniswap uses the Constant Product AMM protocol which is as follows:-
+        * Asset X quantity * Asset Y quantity = K (a constant)  simply  X*Y=K
+          * Properties of popular Constant Product AMM protocol:
+            * Instant liquidity, irrespective of the trade size.
+            * Purchase of asset X increases the price of X and decreases the price of Y
+            * The ratio of assets X and Y sets the price
+        * Now, What is a Protocol?
+          * Protocols are basic sets of rules that allow data and cryptocurrencies to be shared securely between different nodes (i.e computers) in the blockchain network.
+        * Then, What is Liquidity Pool?
+          * A Liquidity Pool is a collection of funds, funded by Liquidity Providers(LP) that are locked in a smart contract, to facilitate trading, borrowing, lending, yielding, etc.
+          * Types of rewards that Liquidity Providers get on funding Liquidity Pools are:
+            * Trading Fees: that are paid by traders on each trade( eg, 0.03% in Curve.fi )
+            - Liquidity Mining:
+              * It is an incentive/reward given to the Liquidity Providers(LP) proportional to the amount of liquidity provided by the LP
+              * It can also be provided to the LPs in terms of “Airdrops”.
+              * Airdrops are tokens that are given to the traders and LPs for free or in exchange for minimal promotional work.
+          - Types of losses for Liquidity Providers:
+            - Impermanent loss == Not Permanent Loss
+              * Loss generated due to price change of the asset which is realized upon withdrawal only!
+              * So, it can result in a total loss of value but:
+                * Trading fees taken from the traders may compensate for it.
+                * Liquidity mining reward may compensate for it. 
+            - Coin de-peg:
+              * De-pegging refers to the phenomenon of a stablecoin deviating or any asset deviating from its intended peg. For example: In the case of a stablecoin pegged to USD, if its value reduces below $1, then the coin is said to be “de-pegged” which indirectly reduces the valuation of investment to a Liquidity Pool by a Liquidity Provider.
+
+          * How traders are making profits using AMM?:-
```

Comment:
we should also make a separate section for Liquidity Pools/Pairs and how do they work